adafruit_funhouse/__init__.py

```diff
@@ -65,17 +71,17 @@ class FunHouse(PortalBase):
     def __init__(
         self,
         *,
-        url=None,
-        headers=None,
-        json_path=None,
-        regexp_path=None,
-        default_bg=0,
-        status_dotstar=None,
-        json_transform=None,
-        rotation=270,
-        scale=1,
-        debug=False,
-    ):
+        url: Optional[str] = None,
+        headers: Dict[str, str] = None,
```

Comment:
Should `headers` type be `Optional[Dict[str, str]]`? since the default value is `None`?

Reply:
Oh, yes, good catch!